src/cli/version.zig

```diff
@@ -36,6 +37,7 @@ pub fn run(alloc: Allocator) !u8 {
     try stdout.print("  - font engine: {}\n", .{build_config.font_backend});
     try stdout.print("  - renderer   : {}\n", .{renderer.Renderer});
     try stdout.print("  - libxev     : {}\n", .{xev.backend});
+    try stdout.print("  - desktop env: {s}\n", .{@tagName(internal_os.desktopEnvironment())});
```

Comment:
I'd only write this out for the GTK apprt. macOS has no others so its just noise.